ops/benton-parity/lib/runtime-probe.sh

```
@@ -0,0 +1,230 @@
+  return 0
+}
+
+# ── Service start probe ───────────────────────────────────────────────────────
+probe_service_start() {
+  local name="$1" project_path="$2" port="$3" health_endpoint="$4"
+
+  log_info "Probing: ${name} (port ${port})"
+  mkdir -p "$RUNTIME_LOG_DIR"
+
+  local full_path="${REPO_ROOT}/${project_path}"
+  local log_file="${RUNTIME_LOG_DIR}/$(echo "$name" | tr ' ()' '___').log"
```

Shellcheck (0.11.0)

[warning] log_file appears unused. Verify use (or export if used externally).

(SC2034)